templates/index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- start fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400&display=swap"
+      rel="stylesheet"
+    />
+    <!-- end fonts -->
+    <link rel="stylesheet" href="../static/css/index.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <!-- start nav -->
+    <nav class="nav" id="nav">
+      <div class="container">
+        <div class="nav__container__parnav">
+          <div class="nav__container__parnav__brand">
+            <a href="#">My Logo</a>
+          </div>
+
+          <div class="nav__container__parnav__bars">
+            <div class="nav__container__parnav__bars__responsive">
+              <img
+                src="..//static/imgs/Union@2x.png"
+                alt="Union@2x"
+                class="responsive nav__container__parnav__bars__responsive__respo"
+              />
+            </div>
+          </div>
+        </div>
+      </div>
+    </nav>
+    <!-- end nav -->
+    <!-- start sectionone -->
+    <section class="sectionone" id="sectionone">
+      <div class="overlay">
+        <div class="sectionone__heroimage">
+          <h2>
+            I’m Alexandra <br />
+            Glad to see you!
+          </h2>
```

Comment:
Kindly make sure to use the font-size and color described in Figma to match the design 🤗
- Font size should be bigger
- Font color should be `172B4D`

Your design:

![image](https://user-images.githubusercontent.com/73714615/191699958-80894014-c5ee-432d-ae39-32c60afd4555.png)

Figma design:

![image](https://user-images.githubusercontent.com/73714615/191700007-e27750ed-dbc9-449a-80aa-95dbba6b72ee.png)